components/layout/index.css

```diff
@@ -33,7 +34,46 @@
   z-index: 1000;
 }
 
-.layout-bar.fixed ~ .layout-sidebar,
-.layout-bar.fixed ~ .layout-main {
+.layout-sidebar,
+.layout-main {
   padding-top: 4.5rem;
 }
+
+.layout-sidebar > .branding {
+  display: none;
+  background: #FAFAFA;
+  padding: 1rem;
+}
+
+@media only screen and (max-width: 800px) {
```

Comment:
You have a variable below but perhaps it does not work. There is `@cutom-media` rule for such things.

Reply:
Variables in @media queries don't work as you said.

I need to set up postcss properly. It's not gonna happen in this PR. I am still waiting once the design package set up is done then I will inspire (copy & paste 😄 ) from there.

Reply:
Good point ☝️